policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/service/PolicyServiceCachingProxy.java

```diff
@@ -69,7 +69,7 @@ public PolicyServiceCachingProxy(final PolicyService service) {
      */
     @CachePut(value = "resourceRules", key = "#resourceId")
     public Optional<Rules<LeafResource>> setResourceRules(final String resourceId, final Rules<LeafResource> rules) {
-        LOGGER.info("Cache add for resourceId {} and rules message {}", resourceId, rules.getMessage());
+        LOGGER.debug("Cache add for resourceId {} and rules message {}", resourceId, rules.getMessage());
```

Comment:
And the same for the set rules log message as well.

Reply:
Added cache name to each